src/50_Samples_%26_Templates/Product.html

```diff
@@ -2,7 +2,7 @@
   "preview": "default",
   "skipValidation": "true",
   "default": "preview",
-  "experiments": ["amp-form-var-sub"]
+  "experiments": ["amp-form-var-sub", "amp-bind"]
   }--->
```

Comment:
amp-selector is already out of experiment. Most likely related to a stale cache. Fixed in #600.